openapi.yaml

```diff
@@ -413,7 +413,7 @@ components:
           type: string
           enum:
             - Blocked
-            - Default
+            - Unblocked
```

Comment:
Do we need to validate these enums when they are submitted to the service?

Reply:
Yea, but this file is wrong anyway.